packages/@mantine/core/src/components/Table/index.ts

```diff
@@ -26,3 +26,10 @@ export type {
   TableTfootProps,
   TableTheadProps,
 } from './Table.components';
+export type { 
+  TableScrollContainerProps,
+  TableScrollContainerStyleNames,
+  TableScrollContainerCssVariables,
+  TableScrollContainerFactory,
+} from './TableScrollContainer';
+export type { TableDataRendererProps } from './TableDataRenderer';
```

Comment:
This should not be exported from the package. TableDataRenderer is an internal component.

Reply:
Removed